puppet/manifests/default.pp

```diff
@@ -0,0 +1,125 @@
+node default {
+
+  include oracle_java
```

Comment:
Probably a separate issue but it's important that `oracle_java` is applied before both `mesos` and `marathon`. For the former, Java is a dep so it will install Java 7 if 8 hasn't been installed yet. For the latter, it will fail to install without Java 8.